components/feature/webnotifications/src/main/java/mozilla/components/feature/webnotifications/NativeNotificationBridge.kt

```diff
@@ -17,17 +19,21 @@ import mozilla.components.browser.icons.IconRequest
 import mozilla.components.browser.icons.IconRequest.Size
 import mozilla.components.concept.engine.webnotifications.WebNotification
 
+private const val EXTRA_ON_CLICK = "onClick"
```

Comment:
Extra should be namespaced and maybe on the companion object.

Reply:
Sure, I'll put it in a companion object